### PR TITLE
use the lower case HEXDIGLC rule in abnf definition

### LINKED ISSUE #219: HEXDIG has uppercase [A-F]
Based on the definition of the HEXDIG from https://tools.ietf.org/html/rfc5234:
HEXDIG =  DIGIT / "A" / "B" / "C" / "D" / "E" / "F"
DIGIT =  %x30-39

It looks to me that the standard requires uppercase. All our examples use lowercase. What is the right thing to do?

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
fixes #219

### CODE REVIEW

**Comment by AloisReitbauer** [comment]
We should be able to simply close this

**Review by AloisReitbauer — APPROVED** [review]

**Review by danielkhan — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
>We should be able to simply close this

close as in merge or decline?

I'll resolve conflict right away

**Comment by SergeyKanzhelev** [comment]
This PR is out 16 days and no concerns so far. With 3 approvals it looks like it's ready to be merged. Should I merge or I should wait for a week after the first approval?

CC: @AloisReitbauer @mtwo @nikmd23

**Review by mtwo — APPROVED** [review]

**Pull request merged**
